app/policies/pageflow/account_policy.rb

```diff
@@ -123,13 +124,13 @@ def destroy_membership_on?
     end
 
     def admin?
-      @user.admin?
+      user.admin?
     end
 
     def see_badge_belonging_to?
-      (@account.entries & @user.entries).any? ||
-        @user.memberships.on_accounts.as_previewer_or_above.where(entity: @account).any? ||
```

Comment:
I think, `FolderPolicy` could also benefit from using the query. Please check if some these `_or_above` methods on `Membership` are unused then and can be removed.